doc/README_Linux.md

```diff
@@ -77,7 +77,7 @@ This method is intended for experienced users, who may want to use opam to insta
   - Debian, Ubuntu:       sudo apt-get install build-essential
   - CentOS, RHEL, Fedora: sudo dnf groupinstall "Development Tools"
   - OpenSuse:             sudo zypper in -t pattern devel_C_C++
-- For CentOS and possibly RHEL some additional steps are required, see [CentOS](#centos) below.
+- For CentOS and possibly RHEL some additional steps are required, see [CentOS](#centos-enable-sudo-for-current-user) below.
```

Comment:
This change is not ok. Instead there should be a section "CentOS" which covers the two subsections.